src/shared-components/globalStyles/style.ts

```diff
@@ -221,6 +221,10 @@ export const brandStyles = (theme: ThemeType) => `
     transition-timing-function: ease-in-out;
   }
 
+  input {
```

Comment:
I think we want to make this selector identical to the one that is causing an issue for iOS 15 Safari: 

```css
input:is([type="button"], [type="submit"], [type="reset"]), input[type="file"]::file-selector-button, button {
    [declarations]
}
```

The `button` part should cover most of our use-cases, but I think to be safe we should copy the entire selector.